galley/pkg/config/analysis/analyzers/injection/injection-version.go

```diff
@@ -49,12 +51,31 @@ func (a *VersionAnalyzer) Metadata() analysis.Metadata {
 		Inputs: collection.Names{
 			collections.K8SCoreV1Namespaces.Name(),
 			collections.K8SCoreV1Pods.Name(),
+			collections.K8SCoreV1Configmaps.Name(),
 		},
 	}
 }
 
 // Analyze implements Analyzer.
 func (a *VersionAnalyzer) Analyze(c analysis.Context) {
+	var injectionVersion string
+	c.ForEach(collections.K8SCoreV1Configmaps.Name(), func(r *resource.Instance) bool {
+		if r.Metadata.FullName.Namespace.String() == "istio-system" &&
```

Comment:
We should add a TODO here that we need some way to specify the namespace the injector is running in.

In theory you could look for it across all namespaces, but I think it's fine to rely on istio-system right now.

Reply:
Added a comment there. The current implementation is using istio-system.